diff -Naur soundtouch.orig/configure.ac soundtouch/configure.ac
--- soundtouch.orig/configure.ac	2014-10-29 22:27:49.000130497 +0400
+++ soundtouch/configure.ac	2014-10-29 22:27:49.003130572 +0400
@@ -21,7 +21,7 @@
 
 AC_INIT(SoundTouch, 1.8.0, [http://www.surina.net/soundtouch])
 AC_CONFIG_AUX_DIR(config)
-AM_CONFIG_HEADER([include/soundtouch_config.h])
+AC_CONFIG_HEADERS([include/soundtouch_config.h])
 AM_INIT_AUTOMAKE
 AM_SILENT_RULES([yes])
 #AC_DISABLE_SHARED	dnl This makes libtool only build static libs 
